c2corg_ui/templates/outing/view.html

```diff
@@ -50,6 +50,9 @@
   });
 
   module.value('documentData', {
+    "document_id": ${outing.get('document_id')},
+    "lang": ${dumps(lang) | n},
```

Comment:
Yes ``lang`` is a string and it should be escaped using ``dumps``, not ``html_entities`` which is the default from the template engine.